defaults:
  - mppi: heijn_push
  - isaacgym: push
  - _self_

render: true
n_steps: 5
nx: 6

actors: ['heijn']
initial_actor_positions: [[0.0, 0.0, 0.0]]

